pkg/cmd/init.go

```diff
@@ -271,6 +278,7 @@ func certSetup(cfg *config.MicroshiftConfig) (*certchains.CertificateChains, err
 					"openshift.default",
 					"openshift.default.svc",
 					"openshift.default.svc.cluster.local",
+					"api-int." + cfg.BaseDomain, // TODO: OpenShift actually uses  api.$ClusterName.$BaseDomain
```

Comment:
Don't we have both for OCP? Do we need both here?

Reply:
Let me add both. I thought api was for the externally serving cert, and api-int for the internally serving. But it would harm nobody.